test/installation/src/util.ts

```diff
@@ -4,29 +4,51 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import type {ExecFileOptions} from 'node:child_process';
-import {execFile as execFileAsync} from 'node:child_process';
+import type {ExecFileException, ExecFileOptions} from 'node:child_process';
+import {execFile as nodeExecFile} from 'node:child_process';
 import {readFile} from 'node:fs/promises';
 import {join} from 'node:path';
-import {promisify} from 'node:util';
 
 import {ASSETS_DIR} from './constants.js';
 
-const nodeExecFile = promisify(execFileAsync);
```

Comment:
This was working strangely and thing were not getting properly returned for stdout/stderr.